ci/util/boot_wait.py

```
@@ -0,0 +1,133 @@
+                    if on_line is not None:
+                        on_line(line)
+                    if time.monotonic() >= deadline:
+                        break
+            except KeyboardInterrupt as ki:
+                from ci.util.global_interrupt_handler import handle_keyboard_interrupt
+
+                handle_keyboard_interrupt(ki)
+                raise
+            except Exception:
+                # Read timeouts are normal.
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)